CHANGELOG.md

```diff
@@ -1,5 +1,35 @@
 # Change Log
 
+## 1.8.0 - 2022-08-19
+
+### Added
+
+- Support for new Python language model [#3235](https://github.com/microsoft/vscode-azurefunctions/pull/3235)
+
+### Changed
+
+- Add net7 tests [#3277](https://github.com/microsoft/vscode-azurefunctions/pull/3277)
```

Comment:
Ah, ok, I was just mimic'ing the style of the versions before; looking further, the style appears to change at some point and the last few versions just listed individual fixes. I personally like the individual fixes in the changelog, as it seems more consistent with additions/changes, and eliminates the need for the user to have to go to another page to find the list of fixes.  That said, it's not a strong opinion either.

Since it appears to be the style of the times, I'll keep it as-is for now.